packages/running/src/index.tsx

```diff
@@ -195,6 +195,10 @@ function Item(props: {
   collapseToggled: ISignal<Section, boolean>;
 }) {
   const { runningItem } = props;
+  // Use a ref instead of a state because the state does not have the time
+  // to update in the callbacks
+  const [collapsed, collapse] = React.useState(false);
```

Comment:
```suggestion
  const [collapsed, setCollapsed] = React.useState(false);
```

The usual React recommendation is to name the two values `[foo, setFoo]`. The `collapse`/`setCollapsed` function never gets called, so this is a minor suggestion.